dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -275,10 +275,8 @@ public FileOutStream createFile(AlluxioURI alluxioPath, CreateFilePOptions optio
       outStreamOptions.setMountId(status.getMountId());
       outStreamOptions.setAcl(status.getAcl());
 
-      FileOutStream ufsOutStream = mDelegatedFileSystem.createFile(ufsFullPath, options);
-
       FileOutStream doraOutStream = mDoraClient.getOutStream(ufsFullPath, mFsContext,
-          outStreamOptions, ufsOutStream, uuid);
```

Comment:
is ufsOutStream still supported? Not find the replacing logic.

Reply:
Yes. Line 283 - 288 is fallback to UFS from client.
The same logic as all other operations.

There is also a switch to control the fallback.

Reply:
Got it.